raiden/storage/sqlite.py

```diff
@@ -440,7 +467,7 @@ def get_snapshot_before_state_change(
 
         cursor = self.conn.execute(
             "SELECT identifier, statechange_qty, statechange_id, data FROM state_snapshot "
-            "WHERE statechange_id <= ? "
+            "WHERE statechange_id <= ? OR statechange_id IS NULL "
```

Comment:
In which case is statechange_id null?

Reply:
In `write_first_state_snapshot` we create the initial state without a state change. That's where we need that condition.